app/Support/Cache/CacheKey.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Support\Cache;
+
+class CacheKey
+{
+    public static function buildUserCanTicketCacheKey(string $userName): string
```

Comment:
Most of the code uses `$username` - is `$userName` more correct? I usually use the former

Reply:
No problem at all, changed to `$username` in https://github.com/RetroAchievements/RAWeb/pull/1659/commits/b8cdfcf5fee4355ad19bbd324d3308c9b032de44.

Reply:
`username` is more correct. It's more well-understood to represent an account. `userName` infers `user name`, which is the user's name, and typically refers to their real name.